pkg/config/config.go

```diff
@@ -38,6 +38,7 @@ const (
 	telemetryTLSVolume = "telemetry-tls"
 	labelsVolume       = "podlabels"
 	annotationsVolume  = "podannotations"
+	podMetaVolume      = "podmeta"
```

Comment:
nit: should this be `podName` since it's mounting the name, not all metadata?

Reply:
Fixed